### PR TITLE
Bump runtime

### PR DESCRIPTION
**Pull request by jkoenig134**
# Readiness checklist

- [ ] I added/updated tests.
- [x] I ensured that the PR title is good enough for the changelog.
- [x] I labeled the PR.
- [x] I self-reviewed the PR.

# Description

https://github.com/nmshd/runtime/compare/6.9.0...6.10.0

Adding password protected X support.

Tests have to be added later.

### CODE REVIEW

**Comment by jkoenig134** [comment]
@Siolto can you add a ticket for next sprint to add simple tests for the additions in this PR?

**Review by aince42 — APPROVED** [review]

File: packages/enmeshed_runtime_bridge/natives/package-lock.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
         "@js-soft/logging-abstractions": "^1.0.1",
         "@js-soft/ts-utils": "^2.3.3",
         "@js-soft/web-logger": "^1.0.4",
-        "@nmshd/app-runtime": "6.9.0",
-        "@nmshd/runtime": "6.9.0",
+        "@nmshd/app-runtime": "6.10.0",
+        "@nmshd/runtime": "6.10.0",
```
Comment (britsta) [inline packages/enmeshed_runtime_bridge/natives/package-lock.json:17]: Can we directly go to version `"6.11.0"` of the Runtime or would that lead to extra work?

Comment (jkoenig134) [reply packages/enmeshed_runtime_bridge/natives/package-lock.json:17]: I'll do that in another PR, that PR will be BIG :D

Comment (britsta) [reply packages/enmeshed_runtime_bridge/natives/package-lock.json:17]: I see :D

**Pull request merged**